src/buttonMaker/RadioAlignText.jsx

```diff
@@ -0,0 +1,20 @@
+import { clickForm } from "./formLogic";
+
+// Radio button for choosing text alignment
+function RadioAlignText({ whereAlign, formDispatch }) {
```

Comment:
question: why is this different from the `RadioBorderStyles`? Couldn't you use one component in both cases?